Libraries/CommonFunctions.psm1

```diff
@@ -2399,4 +2399,32 @@ Function ExtractSSHPublicKeyFromPPKFile {
         } else {
             return $null
         }
+}
+
+function Is-XDPCompatible() {
+    param (
+        [string] $KernelVersion,
+        [string] $DetectedDistro
+    )
+    $MIN_KERNEL_VERSION = "5.6"
+    $RHEL_MIN_KERNEL_VERSION = "4.18.0-213"
+    Write-LogInfo "$KernelVersion $DetectedDistro"
+    # ToDo: Update Minimum kernel version check once patches are in downstream distro.
+    if ($DetectedDistro -eq "UBUNTU"){
```

Comment:
you can refer here to use dict, in case we have more elseif branch when support more distro/kernel version. https://github.com/LIS/LISAv2/blob/a30dcc3b6c12e750bf8fe207b66bced345b623f7/Libraries/CommonFunctions.psm1#L2011